### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by electron0zero**
remove old install steps and link to docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,12 +13,7 @@ This is a Go client for the API used by Grafana's [Synthetic Monitoring
 application](https://github.com/grafana/synthetic-monitoring-app).
 
 For instructions about setting up Synthetic Monitoring in your Grafana
-Cloud instace, please visit [the corresponding
-page](https://grafana.com/grafana/plugins/grafana-synthetic-monitoring-app/installation).
-You will need to generate a [new API
-key](https://grafana.com/profile/api-keys) to initialize the app. You
-can run your own [private
-probes](https://github.com/grafana/synthetic-monitoring-agent).
+Cloud instace, please visit our [docs](https://grafana.com/docs/grafana-cloud/synthetic-monitoring/)
```
Comment (samjewell) [inline README.md:16]: typo
```suggestion
Cloud instance, please visit our [docs](https://grafana.com/docs/grafana-cloud/synthetic-monitoring/)
```

**Review by samjewell — APPROVED** [review]
Seems reasonable to me. Can you briefly say why you're making this change? I'm guessing we want to maintain the docs in a single place and link to that, so we don't have other copies going out of date...?
The part "you will need to generate a new API key" doesn't seem so prominent on the page you're linking to, but maybe that's fine.

**Comment by electron0zero** [comment]
@samjewell 

> Can you briefly say why you're making this change? I'm guessing we want to maintain the docs in a single place and link to that, so we don't have other copies going out of date...?

Yes, only one place for docs, and link to that from everywhere else.

> The part "you will need to generate a new API key" doesn't seem so prominent on the page you're linking to, but maybe that's fine.

API keys are not required anymore, that was for old install flow, so this step is outdated.

**Review by samjewell — APPROVED** [review]
LGTM thanks. Sorry for the delay here - I had intended to approve yesterday, but clicked "comment" instead by accident - my bad.

**Pull request merged**